### DISCUSSION TITLE
Reverse proxy

### DISCUSSION THREAD

**Discussion by TomaszJarkowski** [vercel/next.js#72186] [Help]
### Summary

Hi,

I have a problem because I have an application hosted by vercel, and it works perfectly, a simple application. I would like to connect my vercel application to another application via reverse proxy (my client app) but not to the main path but to some route, for example www.myClientApp.com/vercelApp and currently if I go to this path, i.e. www.myClientApp.com/vercelApp the main page of the vercel application appears correctly the problem is when I want to change the route then I get 308 and redirect and I am immediately transferred to route: www.myClientApp.com/new and I should be on route: www.myClientApp.com/vercelApp/new and here is the problem because I spent many hours on it and I was not able to solve it. I checked many different next.config configurations and manipulated rewrites as well as skipTrailingSlashRedirect but nothing works, it seems to me that this redirect happens before reaching rewrites if someone knows how to help me I would be grateful.
My next.config:

```
module.exports = {
  skipTrailingSlashRedirect: true,
  assetPrefix: 'https://fw-gielda.vercel.app',
  async rewrites() {
    return [
      {
        source: "/:path*",
        destination: "/gielda/:path*"
      }  
    ];
  },
  headers: async () => [
    {
      source: "/:all*(.png|.jpg|.jpeg|.gif|.svg)",
      headers: [
        {
          key: "Cache-Control",
          value: "public, max-age=31536000, must-revalidate",
        },
      ],
    },
  ],
  // crossOrigin: 'use-credentials',
  modularizeImports: {
    '@mui/material': {
      transform: '@mui/material/{{member}}',
    },
    '@mui/lab': {
      transform: '@mui/lab/{{member}}',
    },
  },
  webpack(config) {
    config.module.rules.push({
      test: /\.(mp3)$/,
      use: [
        {
          loader: 'file-loader',
          options: {
            name: 'static/media/[name].[ext]',
          },
        },
      ],
    });
    config.module.rules.push({
      test: /\.svg$/,
      use: ['@svgr/webpack'],
    });
    return config;
  },
};
```

Thanks

**Comment by AbdullahAhmadAAK** [comment]
Did you manage to solve this?
